### PR TITLE
stolostron/backlog#25050 add link to AI reqs (install rev. request)

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: multicluster_engine/install_upgrade/install_connected.adoc [hunk 1]
```diff
@@ -53,6 +53,8 @@ Before you install multicluster engine for Kubernetes, see the following require
 
 ** You must have `cluster-admin` authority to the OpenShift Container Platform Dedicated environment where you are installing the engine.
 
+* If you plan to create managed clusters by using the Assisted Installer, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-on-premise-with-assisted-installer#preparing-to-install-with-ai[Preparing to install with the Assisted Installer] topic in the {ocp} documentation for the requirements.
```
Comment (swopebe) [inline multicluster_engine/install_upgrade/install_connected.adoc]: I don't think we call it this in the doc. Also, I already made a 4.12 update so please check to see if this is covered? (So we don't clutter doc with bullets)

File: multicluster_engine/install_upgrade/install_connected.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ multicluster engine for Kubernetes, you will need to configure an image pull sec
 
 Before you install multicluster engine for Kubernetes, see the following requirements:
 
-* Your RedHat OpenShift Container Platform cluster must have access to the {mce} in the OperatorHub catalog from the OpenShift Container Platform console. 
+* Your Red Hat OpenShift Container Platform cluster must have access to the {mce} in the OperatorHub catalog from the OpenShift Container Platform console.
```
Comment (swopebe) [inline multicluster_engine/install_upgrade/install_connected.adoc:32]: Some of these prereqs are covered by the OCP install. I would request that the install team take a look at this one.

**Review by swopebe — CHANGES_REQUESTED** [review]
couple things, this is an older PR so I wanted to get it moving.

**Comment by cadawson** [comment]
Hi Jakob. Can you take a look at this and make sure that it looks ok from the installation perspective? Thanks!

**Review by JakobGray — APPROVED** [review]

**Review by swopebe — APPROVED** [review]

**Pull request merged**